### DISCUSSION TITLE
Purchasing Options

### DISCUSSION THREAD

**Discussion by ghost** [community/community#60150] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi all,

we have some problems to reach the sales team...

After testing the enterprise on premise version, we would like to buy two licenses, but are trapped in an endless loop.

1. we contacted the sales team via the [website ](https://github.com/enterprise/contact?ref_page=/pricing&ref_cta=Contact%20Sales&ref_loc=cards) and go an automatic reply from sales@github.com.
after that, nothing happens.
2. we send an email directly to sales@github.com to get an automatic reply, that this email is not monitored... (qoute: "Thank you for contacting us! Please note that this email domain ([sales@github.com](mailto:sales@github.com)) is no longer active or monitored.")
but in this email, we find the email address ar@github.com.
3. we send an email to ar@ and receive the answer that we should please contact the sales team via the  [website ](https://github.com/enterprise/contact?ref_page=/pricing&ref_cta=Contact%20Sales&ref_loc=cards)...

this is so confusing and anoying... 

Is there another way to reach out for the sales team?
Or did we miss something?

thank you and kr
Stefan
(LM - IT)

**Comment by GhostOf0days** [comment]
Could you reach out to [Github Support](https://support.github.com/) via a support ticket as they respond? Then they can tell the sales team your issue if they can't fix it. To create a support ticket, click 'My Tickets' in the top right.

**Reply by GhostOf0days** [reply]
Explain to them the sales team issue with the email.

**Reply by ghost** [reply]
did that last week... nothing new so far...

**Comment by queenofcorgis** [comment]
Hi there - Apologies for the confusing experience - I'm looking into this for you.

**Reply by ghost** [reply]
anything new?

**Reply by queenofcorgis** [reply]
Hi there - our sales team did look into this. The email you sent that received a reply was not automatic and a teammate of mine named Claudia should have followed up with you. Our team noticed you had an OoO autoreply to our emails though.

**Reply by ghost** [reply]
Hi there,
yes i had a day off last week, but answered right away on Friday to Claudia

**Reply by queenofcorgis** [reply]
Great - please continue to use that avenue of communication as those folks are best able to assist and will be the most efficient.
